noetic/Dockerfile

```
@@ -0,0 +1,82 @@
+FROM ubuntu:focal
+
+# basic args
+ENV ROS_DISTRO=noetic
+ARG USERNAME=user
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Add the following labels
+LABEL org.opencontainers.image.description="Docker Development Container"
+LABEL org.opencontainers.image.title="ROS1Dev"
```

To maintain consistency across the images
```suggestion
LABEL org.opencontainers.image.title="ROS2DevCon"
```

---

ROS1DevCon maybe?

---

Changed to ROS1DevCon, Refer [7289362](https://github.com/soham2560/DockerForROS2Development/pull/3/commits/728936217944cffb603b659eacb6ec8e3a0ff3d5)